### DISCUSSION TITLE
LFS Media not served correctly

### DISCUSSION THREAD

**Discussion by alexandercerutti** [vercel/community#1787] [Help] [answered]
### Summary

Hi, I have a monorepo of which one folder is a ViteJS project.
I've published that project on a hobby Vercel page. The project works fine.

I have a Github LFS video asset inside that folder, a 350Mb copy of the famous Big Buck Bunny video. If I open the page that loads that asset, the video doesn't get loaded (Firefox returns a MediaError 4, a decoding issue). You can see it here: https://sub37.vercel.app/pages/native-video/index.html

So I started to check what was going on. The video asset that got loaded is actually fine: it is a webm and I can reproduce it correctly. I can also reproduce the one from git.

Therefore I tried to compare the requests by changing the URL in the video tag in the published page, via DevTools:

- GitHub: https://media.githubusercontent.com/media/alexandercerutti/sub37/master/packages/sample/public/bigbuckbunny.webm
- Vercel: https://sub37.vercel.app/bigbuckbunny.webm

I honestly completely ignored the response from Firefox, because it has a reported bug that doesn't decode Base64 in its DevTools, but that was revealed later to be a kind of shortcut to what seemed to happen. Chrome seems to not be able to show content for HTTP 206.

The most noticeable thing I saw was about the HTTP 206 headers `Content-Range` and `Content-Length`.

In fact, Github's link reports:

```
Content-Length: 144434285
Content-Range: bytes 0-144434284/144434285
```

While Vercel's reports:

```
Content-Length: 134
Content-Range: bytes 0-133/134
```

So, I opened ProxyMan and attempted to inspect the request. I found out that the video actually gets served with the following content instead of the real video.

```
version https://git-lfs.github.com/spec/v1
oid sha256:133cefac7d54f968c42da403bc7e95ef9fcb3293d12de81ffaccb97730da9f22
size 144434285
```

Does anyone know if this is a Vercel bug, a Vercel limitation, or something that I didn't configure correctly? I actually have not touched anything in Vercel when I deployed the project, as everything was automatic.

Thank you.

**Comment by alexandercerutti** [comment]
Okay, it is definitely linked to https://github.com/orgs/vercel/discussions/49, but I didn't find anything related to videos so I opened this.

**Comment by amyegan** [comment, marked as answer]
We now support [Git LFS](https://vercel.com/docs/concepts/projects/overview#git-large-file-storage-lfs)! 🎉

**Reply by alexandercerutti** [reply]
Awesome, thank you! Already enabled and redeployed!
